### PR TITLE
Correct devtools usage

### LINKED ISSUE #10389: Review use of "devtools" in Getting Started
### Page URL

https://docs.flutter.dev/platform-integration/android/install-android/install-android-from-linux?tab=virtual

### Page source

/src/content/platform-integration/android/install-android/install-android-from-linux.md

### Describe the problem

This issue applies to this page and many other pages. Recently some pages were updated/added that don't use the terms "DevTools" or "web" correctly.

First, it's either Flutter DevTools, Dart DevTools (both refer to the same suite of tools), or Chrome DevTools. So far as I know, none of these DevTools run on any other browsers besides Chrome, so "web DevTools" is incorrect.

Also, check the Google word list. Unless it's the start of a sentence, it's "web", not "Web".

Finally, "devtools" is not a word. We are talking about two specific sets of tooling. We don't want to weaken the meaning of that term by devolving into a generic short word that could apply to any tooling that is developer related.

Also, regarding the pages that are titled as follows: "Add Android devtools for Flutter from Linux start". I don't even know what this title means (nor any of those pages titled similarly). You don't install DevTools, you get it when you download Flutter/Dart. So you launch it or open it. You do install the Flutter/Dart plugins in your IDE, and maybe you meant that but what is "from a Linux start"? (Another experienced Flutter user told me that they don't understand these titles either).

Also, what are "Android devtools"?

cc @kenzieschmoll 

### Expected fix

_No response_

### Additional context

_No response_

### I would like to fix this problem.

- [ ] I will try and fix this problem on docs.flutter.dev.

### PR DESCRIPTION
**Pull request by atsansone**
Fixes #10389

### CODE REVIEW

File: src/content/platform-integration/android/install-android/index.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Add Android devtools for Flutter
+title: Add Android compiling for Flutter
```
Comment (domesticmouse) [inline src/content/platform-integration/android/install-android/index.md:2]: Compiler tooling?

File: src/content/platform-integration/ios/install-ios/index.md [hunk 2]
```diff
@@ -1,11 +1,12 @@
 ---
-title: Add iOS devtools for Flutter
+title: Add iOS compiler for Flutter
 description: Configure your system to develop Flutter for iOS.
-short-title: Add iOS DevTools
-target-list: [macOS, Android, Web]
+short-title: Add iOS compiling
```
Comment (domesticmouse) [inline src/content/platform-integration/ios/install-ios/index.md:4]: Compiler tooling

File: src/content/platform-integration/android/install-android/index.md (hunk 1 above)
Comment (parlough) [reply src/content/platform-integration/android/install-android/index.md:2]: How about "Set up Android compilation" or "Add Android as a target platform"?

File: src/content/platform-integration/android/install-android/index.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Add Android devtools for Flutter
+title: Add Android as a target platform for Flutter
 description: Configure your system to develop Flutter for Android.
-short-title: Add Android DevTools
-target-list: [Windows, 'Web on Windows', Linux, 'Web on Linux', macOS, 'Web on macOS', iOS, Web on ChromeOS]
+short-title: Add Android compiling
```
Comment (sfshaza2) [inline src/content/platform-integration/android/install-android/index.md:4]: Change to either "Set up Android compilation" or "Configure Android compilation"

File: src/content/platform-integration/android/install-android/install-android-from-ios.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Add Android devtools for Flutter from iOS start
+title: Add Android as a target platform for Flutter from iOS start
```
Comment (sfshaza2) [inline src/content/platform-integration/android/install-android/install-android-from-ios.md:2]: what does "from iOS start" mean? Do you mean "from iOS" or "on iOS"?

File: src/content/platform-integration/android/install-android/install-android-from-linux.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Add Android devtools for Flutter from Linux start
+title: Add Android as a target platform for Flutter from Linux start
```
Comment (sfshaza2) [inline src/content/platform-integration/android/install-android/install-android-from-linux.md:2]: Ditto: "from Linux start". Is that a regional expression?

File: src/content/platform-integration/android/install-android/install-android-from-macos.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Add Android devtools for Flutter from macOS start
+title: Add Android as a target platform for Flutter from macOS start
```
Comment (sfshaza2) [inline src/content/platform-integration/android/install-android/install-android-from-macos.md:2]: Ditto

File: src/content/platform-integration/android/install-android/install-android-from-web-on-chromeos.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Add Android devtools for Flutter from Web on ChromeOS start
+title: Add Android as a target platform for Flutter from Web on ChromeOS start
```
Comment (sfshaza2) [inline src/content/platform-integration/android/install-android/install-android-from-web-on-chromeos.md:2]: Ditto

File: src/content/platform-integration/android/install-android/install-android-from-web-on-linux.md [hunk 8]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Add Android devtools for Flutter from Web on Linux start
+title: Add Android as a target platform for Flutter from Web on Linux start
```
Comment (sfshaza2) [inline src/content/platform-integration/android/install-android/install-android-from-web-on-linux.md:2]: Ditto

File: src/content/platform-integration/android/install-android/install-android-from-web-on-macos.md [hunk 9]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Add Android devtools for Flutter from Web on macOS start
+title: Add Android as a target platform for Flutter from Web on macOS start
```
Comment (sfshaza2) [inline src/content/platform-integration/android/install-android/install-android-from-web-on-macos.md:2]: Ditto

File: src/content/platform-integration/android/install-android/install-android-from-web-on-windows.md [hunk 10]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Add Android devtools for Flutter from Web on Windows start
+title: Add Android as a target platform for Flutter from Web on Windows start
```
Comment (sfshaza2) [inline src/content/platform-integration/android/install-android/install-android-from-web-on-windows.md:2]: Ditto

File: src/content/platform-integration/android/install-android/install-android-from-windows.md [hunk 11]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Add Android devtools for Flutter from Windows start
+title: Add Android as a target platform for Flutter from Windows start
```
Comment (sfshaza2) [inline src/content/platform-integration/android/install-android/install-android-from-windows.md:2]: Ditto

File: src/content/platform-integration/ios/install-ios/index.md (hunk 2 above)
Comment (sfshaza2) [reply src/content/platform-integration/ios/install-ios/index.md:4]: Either "Configure iOS compilation" or "Set up iOS compilation"

File: src/content/platform-integration/web/install-web/index.md [hunk 12]
```diff
@@ -1,11 +1,11 @@
 ---
-title: Add Web devtools for Flutter
+title: Add Chrome DevTools for Flutter
```
Comment (sfshaza2) [inline src/content/platform-integration/web/install-web/index.md:2]: So much better!

**Review by sfshaza2 — COMMENTED** [review]
Much better but still a bit of confusion.

**Comment by sfshaza2** [comment]
Do we use this "from macOS start" (etc) terminology elsewhere? If so, I've never seen it.

File: src/content/platform-integration/android/install-android/install-android-from-ios.md (hunk 4 above)
Comment (atsansone) [reply src/content/platform-integration/android/install-android/install-android-from-ios.md:2]: These pages would most likely be found from their original install page. 

For example, a new user goes to Get Started > Choose OS > macOS choose target > iOS. This is "iOS start".

There's a link at the bottom for them to add Android to their build as a next step, which is why this page says "from iOS start". 

It _could_ say "with iOS target tools already installed", but that seemed like a mouthful!

**Comment by atsansone** [comment]
Added change that Fixes [#10392](https://github.com/flutter/website/issues/10392)

File: src/content/platform-integration/android/install-android/install-android-from-ios.md (hunk 4 above)
Comment (sfshaza2) [reply src/content/platform-integration/android/install-android/install-android-from-ios.md:2]: I still don't like "from iOS start". Why can't it say "on iOS"? Are there multiple "on iOS" pages?

**Review by sfshaza2 — APPROVED** [review]
The comilations headers are better. I still don't like "from iOS start" (etc), but am willing to land for now.

**Pull request merged**